apps/user-office-backend/src/factory/pdf/proposal.ts

```diff
@@ -355,6 +357,14 @@ export const collectProposalPDFData = async (
     }
   }
 
+  // Get Reviews
+  const reviews = await baseContext.queries.review.reviewsForProposal(
+    user,
+    proposal.primaryKey
+  );
+
+  if (reviews) out.sepReviews = reviews;
+
```

Comment:
Looking good but if reviews is returned as an empty array it will pass the if check.

Reply:
@mutambaraf Yeah, that is intentional. If the `reviews` is null, it will fail the if check. But if the `reviews` is [], it passes and assigns the [] to the `out`, which is fine.